pkgs/development/python-modules/cut-cross-entropy/default.nix

```diff
@@ -0,0 +1,68 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytestCheckHook,
+  stdenv,
+  setuptools,
+  setuptools-scm,
+  torch,
+  triton,
+  transformers ? null,
```

Comment:
Not sure, but wouldn't it be better to use `withX ? false` rather than `X ? null` ?

Reply:
Thanks! Changed with this commit (may be squashed): 2cd798d2bdf6a06830b17e61c59108cc6050cd48